src/Engine/Manager/ScriptManager.cpp

```diff
@@ -70,16 +118,9 @@ void AngelScriptDebugLineCallback(asIScriptContext *ctx, const std::map<std::str
     if (breakpoints->find(fileName) != breakpoints->end() && breakpoints->at(fileName).find(line) != breakpoints->at(fileName).end()) {
         // A break point has been reached so the execution of the script should be suspended
         // Show the call stack
-        for (asUINT n = 0; n < ctx->GetCallstackSize(); n++) {
-            asIScriptFunction *func;
-            const char *scriptSection;
-            int line, column;
-            func = ctx->GetFunction(n);
-            line = ctx->GetLineNumber(n, &column, &scriptSection);
-            printf("%s:%s:%d,%d\n", scriptSection,
-                func->GetDeclaration(),
-                line, column);
-        }
+        std::string callstack = CallstackToString(ctx);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'callstack' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)

Reply:
Bumping this. We probably want to print the callstack (and variables) somewhere.

Reply:
I did this initially but because most breakpoints are triggered once or multiple times each frame, it slowed everything down immensely. So i made the decision not to put more time into refining it until we actually need a more roust debugging system. But yeah, maybe we want to log it somewhere?

Reply:
Understood. Seems like a good idea to put it on hold until needed.